src/openms/source/ANALYSIS/ID/AccurateMassSearchEngine.cpp

```diff
@@ -536,18 +553,17 @@ namespace OpenMS
     defaults_.setValue("isotopic_similarity", "false", "Computes a similarity score for each hit (only if the feature exhibits at least two isotopic mass traces).");
     defaults_.setValidStrings("isotopic_similarity", {"false", "true"});
 
-    defaults_.setValue("db:mapping", std::vector<std::string>{"CHEMISTRY/HMDBMappingFile.tsv"}, "Database input file(s), containing three tab-separated columns of mass, formula, identifier. "
+    //TODO should we really support multiple databases? Not sure
```

Comment:
Yeah, I think support for multiple databases would be nice. E.g. on could search for metabolites in HMDB and for additional lipids in Lipid Maps.